Pod/Classes/VSLCall.m

```diff
@@ -52,14 +50,13 @@ @interface VSLCall()
 @property (readwrite, nonatomic) VSLCallTransferState transferStatus;
 @property (readwrite, nonatomic) NSTimeInterval lastSeenConnectDuration;
 @property (strong, nonatomic) NSString *numberToCall;
-@property (weak, nonatomic) VSLAccount *account;
-@property (nonatomic) BOOL reinviteCall;
+//@property (weak, nonatomic) VSLAccount *account;
+@property (nonatomic) BOOL reinviteCall; // Not used, but also never set to NO, so useless?.
```

Comment:
Did you find out what this was used for? If it's not being used then just remove it?

Reply:
This is Arjen's change. Like him, I noticed that `reinviteCall` is just set to `YES` in 2 cases but it is never set to `NO` and never used/checked in the code. SO I think it is safe to remove it. I will remove also the above commented line since the property `account` is declared in the VSLCall.h file. Let me know if anyone has anything to add on this.

Reply:
If Chris verified that it isn't used, I can be removed.

Reply:
Done